src/components/dar_collection_table/DarCollectionTableCellData.js

```diff
@@ -1,6 +1,7 @@
 import {includes, isEmpty, isNil, toLower} from 'lodash/fp';
 import {formatDate} from '../../libs/utils';
-import {h} from 'react-hyperscript-helpers';
+import {h, div, p} from 'react-hyperscript-helpers';
+import { ArrowDropUp, ArrowDropDown } from '@material-ui/icons';
```

Comment:
Are there standardized or better drop up / drop down icons ? I used these (as I did somewhere else) but I am not sure they are the most attractive. I couldn't find other usages of drop up/drop down icons in the codebase.

Reply:
I don't think we do have something like that. Perhaps run it by Yvonne for her thoughts.

Reply:
I found one, used in the code + looks like the mock.